cli/aws_orbit/remote_files/kubectl.py

```diff
@@ -337,7 +338,7 @@ def fetch_kubectl_data(context: "Context", k8s_context: str, include_teams: bool
     )
 
     context.landing_page_url = f"https://{landing_page_url}"
-    context.k8_dashboard_url = f"http://{k8_dashboard_url}"
+    context.k8_dashboard_url = f"https://{k8_dashboard_url}"
```

Comment:
We don't expose the k8_dashboard via the elb, so I think for now it needs to stay http